stock_picking_cancel_confirm/wizard/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import cancel_confirm
```

Comment:
**In my option, we don't need to patch any of this wizard.**
It was the logic of Odoo core in picking.action_cancel() that do not cancel picking (whether it is a bug or the intention)

BUT, if we need to fix it we should fix at the right place, which is in action_cancel() itself.